__tests__/__snapshots__/runner.js.snap

```diff
@@ -1425,9 +1431,7 @@ export class PetshopApi {
      * @method
      * @name PetshopApi#getInventory
      */
-    getInventory(parameters: {} & CommonRequestOptions): Promise < ResponseWithBody < 200, {
-        [key: string]: number
-    } >> {
+    getInventory(parameters: {} & CommonRequestOptions): Promise < ResponseWithBody < 200, object >> {
```

Comment:
this will be fixed/enhanced in a separate PR dealing with pre-defined `response` definitions...